pkg/backup/main.go

```diff
@@ -108,6 +108,7 @@ func DoBackupCreate(volumeName, snapshotName, destURL, backingImageName, backing
 	volume := &backupstore.Volume{
 		Name:             volumeName,
 		Size:             volumeInfo.Size,
+		Labels:           labelMap,
```

Comment:
Technically this is not needed since the labels come from the deltaConfig below.
But it's a good idea none the less since by assigning this ensure that a volume has a status while the initial backup is in progress.